bin/matrix-list.js

```diff
@@ -30,12 +30,24 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
       });
     } else if (target.match(/app/)) {
       Matrix.validate.device(); //Make sure the user has selected a device
-      Matrix.firebase.app.list(function (err, data) {
+      Matrix.firebase.app.list(function (err, apps) {
         Matrix.loader.stop();
         if (err) return console.error('- ', t('matrix.list.app_list_error') + ':', err);
-        if (_.isUndefined(data)) data = {};
-        console.log(Matrix.helpers.displayApps(data));
-        process.exit();
+        if (_.isUndefined(apps)) apps = {};
+
+        //Retrieve status for each app
+        async.forEach(Object.keys(apps), function (appId, done) { 
```

Comment:
I don't understand. Shouldn't `meta.status` be available on `apps`? Why are we doing all these lookups?